mmdet/apis/ote/apis/detection/openvino_task.py

```diff
@@ -158,6 +171,21 @@ def forward(self, inputs: Dict[str, np.ndarray]) -> Dict[str, np.ndarray]:
         return self.model.infer(inputs)
 
 
+class ScOpenVinoDataLoader(DataLoader):
```

Comment:
Please use 'OTE' or anything instead of 'SC'. And in general try to avoid references to 'SC'.

Reply:
Ok, fixed